js/renovation/widget.tsx

```diff
@@ -16,6 +16,7 @@ import { each } from '../core/utils/iterator';
 import { extend } from '../core/utils/extend';
 import { focusable } from '../ui/widget/selectors';
 import { isFakeClickEvent } from '../events/utils';
+import { useLayoutEffect } from 'preact/hooks';
```

Comment:
We shouldn't use any react/preact things. Why Effect decorator is not used here?